packages/office-ui-fabric-react/src/components/pickers/examples/TagPicker.Basic.Example.tsx

```diff
@@ -3,14 +3,26 @@ import * as React from 'react';
 import { TagPicker, IBasePicker, ITag } from 'office-ui-fabric-react/lib/Pickers';
 import { Checkbox, ICheckboxStyles } from 'office-ui-fabric-react/lib/Checkbox';
 import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+import { useBoolean, useConstCallback } from '@uifabric/react-hooks';
 
 const rootClass = mergeStyles({
   maxWidth: 500,
 });
 
 const checkboxStyles: Partial<ICheckboxStyles> = { root: { margin: '10px 0' } };
 
-const _testTags: ITag[] = [
+const inputProps = {
+  onBlur: (ev: React.FocusEvent<HTMLInputElement>) => console.log('onBlur called'),
+  onFocus: (ev: React.FocusEvent<HTMLInputElement>) => console.log('onFocus called'),
+  'aria-label': 'Tag Picker',
+};
+
+const pickerSuggestionsProps = {
+  suggestionsHeaderText: 'Suggested Tags',
+  noResultsFoundText: 'No Color Tags Found',
```

Comment:
While you're here, these should be sentence case.
```suggestion
  'aria-label': 'Tag picker',
};

const pickerSuggestionsProps = {
  suggestionsHeaderText: 'Suggested tags',
  noResultsFoundText: 'No color tags found',
```